doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,142 @@
+
+1. Select 1000 samples for the control period, and 2 x 1000 samples for the future period (one for each subscenario). Step 1 poses a constraint on winter precipitation. For the control period, winter precipitation must still closely represent the average of the original ensemble. For the two future periods, the change in winter precipitation with respect to the control period must approximately equal :math:`4{\Delta}T` % (subscenario L) or  :math:`8{\Delta}T` % (subscenario H).
+2. Further constrain the selection by picking samples that represent either high or low changes in summer precipitation and summer and winter temperature, by limiting the remaining samples to certain percentile ranges: relatively wet/cold in the control and dry/warm in the future, or vice versa. The percentile ranges are listed in table 1 of Lenderink 2014's supplement. This should result is approximately 50 remaining samples for each scenario, for both control and future.
+3. Use a monte-carlo method to make a final selection of 8 resamples with minimal reuse of the same ensemble member/segment.
+
+
+.. note::
+    We highly recommend using the options described in :ref:`rerunning`. The speed bottleneck for the first diagnostic is the preprocessor. In the second diagnostic, step 1 is most time consuming, whereas steps 2 and 3 are likely to be repeated several times. Therefore, intermediate files are saved after step 1, and the diagnostic will automatically detect and use them if the :code:`-i` flag is used.
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in recipes/
+
+- recipe_kcs.yml
+
+Diagnostics are stored in diag_scripts/kcs/
+
+- global_matching.py
+- local_resampling.py
+
+
+User settings
+-------------
+
+Datasets: Datasets have been split in two parts, such that the CMIP ensemble and the target model datasets can be treated separately. The recipe can work with a target model that is not part of CMIP. In that case the target model data should be stored alongside the CMIP data in the exact same format. One use case for this recipe is to compare between CMIP5 and CMIP6.
+
+Preprocessors: We've tried to use built-in preprocessor functions as much as possible. The first diagnostic requires global mean temperature anomalies for each dataset, both CMIP and the target model, and some multimodel statistics. The second diagnostic requires focuses on a point in the Netherlands. The :code:`extract_point` preprocessor can be changed to :code:`extract_shape` or :code:`extract_region`, in conjunction with an area mean. And of course, the coordinates can be changed to analyze a different region.
+
+Diagnostics:
+
+* global_matching
+
+  * :code:`scenario_years`: a list of time horizons. Default: :code:`[2050, 2085]`
+  * :code:`scenario_percentiles`: a list of percentiles for the steering table. Default: :code:`[p10, p90]`
+  * :code:`target_model`: the name of the target model. Default: :code:`EC-Earth`
```

Comment:
shall we change the default EC-Earth to CCSM4, because the plots show the results for CCSM4?

Reply:
Hmm. Maybe I was hoping to fix the EC-Earth data that are part of CMIP5, so we can use that instead.

Reply:
I added a comment about the target model. We cannot just change it without the comment, because all the values have been tuned to ec-earth. So I think, without a good EC-Earth CMIP5 entry, we cannot do better than this.